apps/extension/src/utils/constants/side.ts

```diff
@@ -1,2 +1,50 @@
 export const MIN_SIDE_CONTENT_WIDTH = 400 // px
 export const DEFAULT_SIDE_CONTENT_WIDTH = 400 // px
+
+export const DOWNLOAD_FILE_ITEMS = {
+  md: {
+    label: 'Markdown',
+  },
+}
+
+// 段落深度
+export const PARAGRAPH_DEPTH = 3
+
+export enum MARKDOWN_TEMPLATE_TOKEN {
```

Comment:
`Read Frog` can be derived from `APP_NAME`, don't hard code it. then if you change the app name, all places will be changed.